packages/core/src/odata/common/request-builder/batch/batch-response-transformer.ts

```diff
@@ -0,0 +1,220 @@
+import { createLogger } from '@sap-cloud-sdk/util';
```

Comment:
Minor: To keep our general notion would the name `Batch-Response-Deserializer` not be more appropriate?

Reply:
True